src/dos/dos.cpp

```diff
@@ -148,7 +149,122 @@ static uint16_t DOS_GetAmount(void) {
 #ifndef DOSBOX_CPU_H
 #include "cpu.h"
 #endif
-static inline void modify_cycles(Bits value) {
+
+int hdd_data_rate_byte_per_sec   = 0;
+int fdd_data_rate_byte_per_sec   = 0;
+int cdrom_data_rate_byte_per_sec = 0;
+bool fdd_delay_enabled		 = false;
+bool hdd_delay_enabled		 = false;
+bool cdrom_delay_enabled	 = false;
```

Comment:
Could you wrap all these in a global struct to "namespace them a little"? Like `struct MixerSettings` in `mixer.cpp`. Then instantiate a static of that `struct DiskSpeed` or something.